### PR TITLE
i18n(es): Update install/manual

### PR DESCRIPTION
**Pull request by glopzel**
#### What kind of changes does this PR include?

- New or updated content
- Translated content
- Changes to the docs site code

#### Description

- What does this PR change? adds create tsconfig.json section and updates the file structure at section 7

### CODE REVIEW

File: src/pages/es/install/manual.md [hunk 1]
```diff
@@ -145,7 +145,26 @@ Si deseas incluir [componentes de frameworks](/es/core-concepts/framework-compon
...
+``` json title="tsconfig.json" "base"
+{
+	"extends": "astro/tsconfig/base",
+	"compilerOptions": {
+		"types": ["astro/client"]
+	}
+}
+```
+
+Lee nuestra [guía para configurar Typescript](es/guides/typescript#configuración) para más información.
```
Comment (agustinmulet) [inline src/pages/es/install/manual.md:165]: I think this should fix the link 😄 

```suggestion
Lee nuestra [guía para configurar Typescript](/es/guides/typescript#configuración) para más información.
```

File: src/pages/es/install/manual.md [hunk 2]
```diff
@@ -145,7 +145,26 @@ Si deseas incluir [componentes de frameworks](/es/core-concepts/framework-compon
 
 📚 Lee nuestra guía de [configuración de API](/es/reference/configuration-reference/) para más información.
 
-## 6. Siguientes pasos
+## 6. Crea `tsconfig.json`
+
+Typescript es configurado usando `tsconfig.json`. Aún si tu no escribes código en TypeScript, este archivo es importante para que herramientas como Astro y VS Code sepan como comprender tu proyecto. Algunas caracteristicas (como importaciones de paquetes npm) no tienen un soporte completo en el editor sin el archivo `tsconfig.json`.
```
Comment (agustinmulet) [inline src/pages/es/install/manual.md:150]:
```suggestion
Typescript es configurado usando `tsconfig.json`. Aún si tú no escribes código en TypeScript, este archivo es importante para que herramientas como Astro y VS Code sepan cómo comprender tu proyecto. Algunas características (como importaciones de paquetes npm) no tienen un soporte completo en el editor sin el archivo `tsconfig.json`.
```

File: src/pages/es/install/manual.md [hunk 3]
```diff
@@ -145,7 +145,26 @@ Si deseas incluir [componentes de frameworks](/es/core-concepts/framework-compon
...
+
+Si pretendes escribir código en TypeScript, usar las plantillas de Astro `strict` o `strictest` es recomendado. Tu puedes ver y comparar las tres configuraciones de las plantillas en [astro/tsconfigs/](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/).
```
Comment (agustinmulet) [inline src/pages/es/install/manual.md:152]:
```suggestion
Si pretendes escribir código en TypeScript, recomendamos usar las plantillas de Astro `strict` o `strictest`. Puedes ver y comparar las tres configuraciones de las plantillas en [astro/tsconfigs/](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/).
```

File: src/pages/es/install/manual.md [hunk 4]
```diff
@@ -145,7 +145,26 @@ Si deseas incluir [componentes de frameworks](/es/core-concepts/framework-compon
...
+
+Crea `tsconfig.json` en la raíz del proyecto, y copia el siguiente código en el archivo. (Puedes usar `base`, `strict` o `strictest` para tu maquetado de TypeScript):
```
Comment (agustinmulet) [inline src/pages/es/install/manual.md:154]:
```suggestion
Crea `tsconfig.json` en la raíz del proyecto y copia el siguiente código en el archivo. (Puedes usar `base`, `strict` o `strictest` para tu maquetado de TypeScript):
```

**Review by agustinmulet — COMMENTED** [review]
Pequeñas sugerencias, muy buen trabajo! 💪

**Review by agustinmulet — APPROVED** [review]
LGTM! ✅

**Review by delucis — APPROVED** [review]
Great work! Thanks @glopzel & @agustinmulet 🚀

**Pull request merged**